PORTAL/jobs/_utils.py

```diff
@@ -3098,7 +3106,7 @@ class PlatformInfo(namedtuple('PlatformInfo', 'kernel')):
              )
          )
         '''
-    REGEX = re.compile(rf'^({KERNEL})$', re.VERBOSE)
+    REGEX = re.compile(rf'^({KERNEL})$', re.VERBOSE)  # noqa: F821
```

Comment:
What's the problem here?

Reply:
I think this is a flake8 bug -- it things `KERNEL` isn't defined, since it's in class scope (I think).  I confirmed it's working as one would expect here, so I'm just suppressing that error.  (That error is otherwise *really* useful in function bodies, so I don't want to turn it off globally).

Reply:
Yeah, resolution of class attrs defined earlier in a class definition has some weird caveats so I can see how flake8 might get it wrong (or try to discourage that usage).  +1 to suppressing the error.